docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+    api-dummy-fpm:
+        build:
+            context: ./api-dummy
+        volumes:
+            - ./api-dummy:/srv/app
```

Comment:
The `README` in the root directory needs some basic instruction thought, explaining what `docker-compose up` will do.